recipe/meta.yaml

```diff
@@ -24,6 +24,7 @@ requirements:
     - pip
   run:
     - python
+    - tomli
```

Comment:
Just a comment like `# extra dep` at the top here will do.

Reply:
:+1:  Done